### ISSUE TITLE
Image stretching in universal modules carousel template seems not work in all cases

### ISSUE THREAD

**Issue by micker** [FLEXIcontent/flexicontent-cck#336]
i think stretch image option in carrousel module seems to be broken
http://com3elles.org/concept-sonore.com/index.php?option=com_content&view=featured&Itemid=1347
first right box (actu1)

**Comment by ggppdk** [comment]
"Stretch" is maybe not best word for this maybe we need to rename parameter option

About the issue
We add this inline CSS to the image:
```
width: auto; max-width: 100%
height: auto; max-height: 100%
```

instead we need (will make fix):
```
width: 100%; max-width: 100%
height: auto; max-height: 100%
```

Unfortunately we can not use CSS3:
```
object-fit: contain;
```

if we could have used: object-fit
then we would be able to more easily add other options to the image fit
- crop equally (left and right, top and bottom )
-

**Comment by ggppdk** [comment]
And also we need to change the CSS of the div container if the image

**Comment by ggppdk** [comment]
- Fixed with commit a98cf30
(remember must enable image "cleared" or "overlayed"),

- also fixed height calculation on browser windows resizing

- and continue CSS work of @micker on 

a. making carousel boderless and paddless by default (this was already partly possible via params)

b. navigation to be inside carousel (improved CSS of it be more accessible)

c. also the active / hovered item highlighting is now optional via parameters and by default it is OFF
